src/test/java/com/algorand/algosdk/integration/Stepdefs.java

```diff
@@ -118,6 +119,31 @@ public class Stepdefs {
     Response<CompileResponse> compileResponse;
     Response<DryrunResponse> dryrunResponse;
 
+    private static class DevModeState {
+        static final long ACCOUNT_FUNDING_MICROALGOS = 100_000_000;
+        private Account advanceRounds;
+
+        /**
+         * randomAmount minimizes the chance `advanceRounds` issues duplicate transactions by randomizing the payment amount.
```

Comment:
@winder Resolving with the accompanying note - let me know if you'd like to discuss further.

* Agree _note_ can also be used.
* I don't have a strong rationale here for preferring random note vs amount.  I'd like to dissuade any SDK maintainer from trying to assert on `advanceRounds` funding levels.  Since it seems you don't feel strongly, I'll opt to retain as is for now.